lib/utils.js

```diff
@@ -59,4 +60,36 @@ async function resolveExecutablePath (cmd) {
   return null;
 }
 
-export { pkgRoot, ok, nok, okOptional, nokOptional, inquirer, configureBinaryLog, authorizeIos, resolveExecutablePath};
+/**
+ * @typedef {Object} NpmPackageInfo
+ * @property {string} version - version
+ * @property {string} path - A path to npm root
+ */
+/**
+ * Returns the path and version of given package name
+ * @param {string} packageName A package name to get path and version data
+ * @return {?NpmPackageInfo}
+ */
+async function getNpmPackageInfo (packageName) {
+  const npmPath = await resolveExecutablePath(`npm${system.isWindows() ? `.cmd` : ''}`);
+  if (!npmPath) {
+    return nokOptional(`'npm' binary not found in PATH: ${process.env.PATH}`);
+  }
+
+  let pJson = {};
+  try {
+    const {stdout} = await exec(npmPath, ['list', '-g', '-l', '-j', packageName]);
+    pJson = JSON.parse(stdout);
+  } catch (err) {
+    log.debug(err);
```

Comment:
return null?

Reply:
Could reach to the last `null` so I skipped that. Will add it as safer way.